### PR TITLE
Added Wishlist Component

### PR DESCRIPTION
**Pull request by aniketxparihar**
Pages Added
Wishlist
Components Added
WishlistCard

### CODE REVIEW

File: src/components/WishlistCard/WishlistCard.jsx [hunk 1]
```diff
@@ -1,20 +1,90 @@
...
+      {
+        headers: {
+          authorization: authToken,
+        },
+      }
     );
-}
+    removeFromWishlist(product);
+  };
+  console.log("Hello", product.product);
```
Comment (Vikas2426) [inline src/components/WishlistCard/WishlistCard.jsx:33]: should remove console.log

File: src/pages/Wishlist/Wishlist.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    padding-right: auto;
+    gap: 4rem;
+}
+
+.wishlist__products::-webkit-scrollbar {
+    display: none;
+}
+
+
+
```
Comment (shraddha15-creator) [inline src/pages/Wishlist/Wishlist.css:34]: remove empty lines

File: src/pages/Wishlist/Wishlist.jsx [hunk 3]
```diff
@@ -1,20 +1,41 @@
...
+import React, { useState, useEffect } from "react";
+import "./Wishlist.css";
+import { useWishlist } from "../../Context/wishlist-context";
+import getDiscount from "../../components/Discount/Discount";
+import { useAuth } from "../../Context/auth-Context";
 import axios from "axios";
-import ProductCard from '../../components/ProductCard/ProductCard';
-import WishlistCard from '../../components/WishlistCard/WishlistCard';
-
+import ProductCard from "../../components/ProductCard/ProductCard";
```
Comment (Vikas2426) [inline src/pages/Wishlist/Wishlist.jsx:7]: add an index.js file in your components folder and import your components from there

File: src/pages/Wishlist/Wishlist.jsx [hunk 4]
```diff
@@ -1,20 +1,41 @@
-import React,{useState,useEffect} from 'react'
-import "./Wishlist.css"
-import { useWishlist } from '../../Context/wishlist-context';
-import getDiscount from '../../components/Discount/Discount';
-import { useAuth } from '../../Context/auth-Context';
+import React, { useState, useEffect } from "react";
+import "./Wishlist.css";
+import { useWishlist } from "../../Context/wishlist-context";
+import getDiscount from "../../components/Discount/Discount";
+import { useAuth } from "../../Context/auth-Context";
```
Comment (shraddha15-creator) [inline src/pages/Wishlist/Wishlist.jsx:5]: export useAuth and useWishlist in index.js, then import it from there

**Review by Vikas2426 — APPROVED** [review]

**Review by shraddha15-creator — APPROVED** [review]
Code is readable 
Good that you keep a PR small

**Pull request merged**